packages/coverage-istanbul/src/provider.ts

```diff
@@ -396,16 +401,29 @@ export class IstanbulCoverageProvider
       .filter(file => !coveredFiles.includes(file))
       .sort()
 
-    const cacheKey = new Date().getTime()
     const coverageMap = libCoverage.createCoverageMap({})
 
+    // Make sure file is not served from cache so that instrumenter loads up requested file coverage
+    const cacheKey = new Date().getTime()
+
     // Note that these cannot be run parallel as synchronous instrumenter.lastFileCoverage
     // returns the coverage of the last transformed file
     for (const [index, filename] of uncoveredFiles.entries()) {
       debug('Uncovered file %s %d/%d', filename, index, uncoveredFiles.length)
 
-      // Make sure file is not served from cache so that instrumenter loads up requested file coverage
-      await this.ctx.vitenode.transformRequest(`${filename}?v=${cacheKey}`)
+      for (const vitenode of viteNodeServers) {
+        try {
+          await vitenode.transformRequest(`${filename}?v=${cacheKey}`)
+          continue
```

Comment:
do you want to continue or do you want to break here? `continue` doesn't do anything here

Reply:
ohh it should be `break` 🤦 
Good catch!